template/dockerfiles/erlang/draft.yaml

```diff
@@ -5,4 +5,15 @@ nameOverrides:
 variables:
   - name: "PORT"
     description: "the port exposed in the application"
-    type: int
\ No newline at end of file
+    type: int
+  - name: "BUILDERVERSION"
+    description: "the version of erlang used by the application"
+  - name: "VERSION"
+    description: "the version of alpine used by the application"
```

Comment:
it could be helpful to have a slightly different description for the BUILDERVERSION